sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -12,7 +12,7 @@
     <EnableClientSdkAnalyzers>false</EnableClientSdkAnalyzers>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'net5.0'">
```

Comment:
just to make sure I understand, we are excluding these packages when building against 5.0 because they are already included there with the version/functions we need, right?

Reply:
Correct.